### DISCUSSION TITLE
Cannot build Sway for RPi4B with buildarmimg

### DISCUSSION THREAD

**Discussion by Lightwel** [Manjaro-Sway/manjaro-sway#513] [Q&A] [answered]
Help please - having a lot of trouble with Sway.

I'm trying to build the manjaro-sway edition for RaspberryPi 4B.

cloned the git-hub repository

git clone https://manjaro-sway/manjaro-sway.git

cd manjaro-sway

sudo buildarmimg -d rpi4 -e sway 

(note buildiso is not available in the aarch64 repository - could be useful), and manjaro-sway-22.0.4-230430-linux61.iso downloaded from Manjaro Downloads will install but won't boot - see my post at https://forum.manjaro.org/t/sway-version-stuck-on-first-start-no-ideas-left/139960.

During the build I get the following error

Cannot stat /var/lib/manjaro-arm-tools/tmp/root/boot/* No such file or directory

and though the flashed image produces both BOOT_MNJARO and ROOT_MNJARO, /boot is empty and so is /root/boot.

I have used the same process with the Gnome edition.  The flashed image produces a populated /boot partition without the need to extract files from /root/boot.   It boots to desktop, but I cannot get into any apps (including terminal) or TTY2.  For now I am leaving it there.

I have built the Sway image twice with the above buildarmimg command with the same result. 

What am I doing wrong?

Tks
R

**Comment by boredland** [comment]
This repository is not supporting the arm edition. Please head over to the Manjaro forum.

**Reply by Lightwel** [reply]
Thanks for the reply.  Will do.

R

**Reply by boredland** [reply, marked as answer]
this repository now supports arm, currently only for the rpi4 (which should work on rpi4 and newer) images. You may give i a try: https://manjaro-sway.download
Feedback is appreciated and collected [here](https://github.com/manjaro-sway/manjaro-sway/issues/712)
